cli/medperf/comms/interface.py

```diff
@@ -40,30 +39,6 @@ def authenticate(self):
         """Retrieve a token stored locally for authentication
         """
 
-    @abstractmethod
-    def benchmark_association(self, benchmark_uid: int) -> Role:
-        """Retrieves the benchmark association
-
-        Args:
-            benchmark_uid (int): UID of the benchmark
-
-        Returns:
-            Role: the association type between current user and benchmark
-        """
-
-    @abstractmethod
-    def authorized_by_role(self, benchmark_uid: int, role: str) -> bool:
```

Comment:
Any reason why we're removing this? I don't remember where we use it, but I'd like to know the reasoning behind this

Reply:
yeah, there is no reason. Just wanted to remove unused rest methods. lmk if I should revert that back